### ISSUE TITLE
Modal Loses functionality at 400% zoom - The email opt-in modal

### ISSUE THREAD

**Issue by MajoMaldonado** [equalizedigital/accessibility-checker#1779]
Location / URL: https://edac-vpat.instawp.co/wp-admin/admin.php?page=accessibility_checker 
Screenshot: 

<img width="632" height="217" alt="Image" src="https://github.com/user-attachments/assets/b8a43c01-1af4-4031-8df1-c78a86af2b57" />

WCAG S.C: 1.4.10 Reflow (Level AA)
Issue Description: Preconditions: Set the display resolution to 1920×1080 and the display scale to 100% in the operating system settings. Then, set the browser zoom level to 400%.

When the browser is zoomed to 400%, the modal expands beyond the viewport and does not reflow to fit the available screen space. As a result, portions of the modal become inaccessible, preventing users from accessing all content and controls. This causes the modal to lose functionality, as users cannot fully interact with or complete the intended task without content being obscured or unreachable.
